stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -137,6 +137,7 @@ function versioninfo(io::IO=stdout; verbose::Bool=false)
     println(io, "  WORD_SIZE: ", Sys.WORD_SIZE)
```

Comment:
Semi-relatedly, it may be interesting to alter the CPU printing to add `length(cpu)` to the output:
```
  CPU: 8 × Intel(R) Core(TM) i7-8559U CPU @ 2.70GHz
```

(with unicode `×`)

Reply:
Done!